File: TEMA1/python/mayormenor.py
def mayor (a, b, c):
    if (a> b and a > c) :
        return f'"Mayor : {a}'
    elif (b > a and b > c):
        return f'"Mayor : {b}"'
    else: 
        return f' "Mayor : {c}"'
def menor (a, b, c):
    if (a< b and a < c) :
        return f'"Menor : {a}'
    elif (b < a and b <c):
        return f'"Menor : {b}"'
    else: 
        return f' "Menor : {c}"'


print ("****MAYOR MENOR****")
print("Introduzca el valor de a:")
a = int (input())
print("Introduzca el valor de b:")
b = int (input())
print("Introduzca el valor de c:")
c = int (input())
print(mayor(a,b,c))
print(menor(a,b,c))

File: TEMA1/typescript/mayormenor.ts
import readline from "node:readline/promises";
import { stdin as input, stdout as output } from "node:process";
const rl = readline.createInterface({ input, output }); 

function mayor(a: number, b: number, c: number){
if(a > b && a > c) return `Mayor: ${a}`;
else if(b > a && b > c) return `Mayor: ${b}`;
else return `Mayor: ${c}`;
}

function menor(a: number, b: number, c: number){
if(a < b && a < c) return `Menor: ${a}`;
else if(b < a && b < c) return `Menor: ${b}`;
else return `Menor: ${c}`;
}

console.log("********Mayor Menor**********")
const a = Number (await rl.question("Introduzca el valor de a: "))
const b = Number (await rl.question("Introduzca el valor de b: "))
const c = Number (await rl.question("Introduzca el valor de c: "))

console.log(mayor(a,b,c));
console.log(menor(a,b,c));
rl.close();
